packages/core/src/middleware/koa-user-log.ts

```diff
@@ -1,14 +1,23 @@
-import { UserLogPayload, UserLogResult, UserLogType } from '@logto/schemas';
+/** TODO: remove deprecated code after removing all usages (LOG-2220), and rename to koa-log */
```

Comment:
As your PR summary described, koa-user-log should be removed and a new koa-log is added.  This is confusing, what changes exactly this PR should cover?